.env.example

```
@@ -1,95 +1,169 @@
+# Required: GitHub webhook secret (set in GitHub organization settings)
+GITHUB_WEBHOOK_SECRET=your_webhook_secret_here
+
+# Webhook server port (different from main MCP server)
+WEBHOOK_PORT=38080
+
+# Repository filtering (* for all, or comma-separated list: "owner/repo1,owner/repo2")
+REPOS_TO_TRACK=*
+
+# Webhook processing configuration
+WEBHOOK_SCRIPT_PATH=./scripts/extract_coderabbit_prompts.py
+WEBHOOK_OUTPUT_DIR=./webhook_outputs
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The WEBHOOK_OUTPUT_DIR key should go before the WEBHOOK_SCRIPT_PATH key

(UnorderedKey)